assets/scss/styles.scss

```diff
@@ -208,6 +193,10 @@ h1 {
             border-bottom: 2px solid #0b0c0c;
```

Comment:
These colours are actually part of the gov.uk palette, and actually have variables assigned, in this instance, it's $black.